Providers/nxOMSAutomationWorker/automationworker/worker/omsutil.py

```diff
@@ -79,6 +79,16 @@ def initialize():
     else:
         print "Successfully set owners of certificate folder."
 
+    proxy_paths = ["/etc/opt/microsoft/omsagent/conf/proxy.conf", "/etc/opt/microsoft/omsagent/proxy.conf"]
+    for path in proxy_paths:
+        if os.path.exists(path):
+            process, output, error = linuxutil.popen_communicate(["sudo", "chmod", "g+r", path])
```

Comment:
Why do you need to add this permission? The proxy.conf is intended to have 600 perms for omsagent:omiusers (see [omsadmin.sh](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/installer/scripts/omsadmin.sh#L294))
I'm just looking for clarity on why the need for this permission hasn't come up earlier.